src/ui/modals/confirm_box.rs

```diff
@@ -0,0 +1,77 @@
+use ::tui::buffer::Buffer;
+use ::tui::layout::Rect;
+use ::tui::style::{Color, Modifier, Style};
+use ::tui::widgets::Widget;
+
+use crate::ui::format::truncate_middle;
+use crate::ui::grid::draw_filled_rect;
+
+fn render_confirm_prompt(buf: &mut Buffer, confirm_rect: &Rect, confirm_msg: &str) {
+    let text_style = Style::default()
+        .bg(Color::Black)
+        .fg(Color::White)
+        .modifier(Modifier::BOLD);
+
+    let text_max_length = confirm_rect.width - 4;
+    let confirm_text = truncate_middle(confirm_msg, text_max_length);
```

Comment:
Right now, what happens with this line is that the "Are you sure you want to quit?" line gets truncated on a small window to something like "Are you su[...] to quit?"
Do you think we can have a few options here and make it "shorten" itself if the window is not wide enough?
eg. "Are you sure you want to quit?" ==> "Sure you want to quit?" ==> "Really quit?" ==> "Quit?"
There are a few examples of this in the code, like here: https://github.com/imsnif/diskonaut/blob/main/src/ui/term_too_small.rs#L16-L33

Reply:
Sure. i think i can handle that with the explanation you just did.
I'll inform you in case of any problem.